Detectors/MUON/MCH/Workflow/src/TimeClusterFinderSpec.cxx

```diff
@@ -170,10 +182,10 @@ o2::framework::DataProcessorSpec
     outputs,
     AlgorithmSpec{adaptFromTask<TimeClusterFinderTask>()},
     Options{{"mch-debug", VariantType::Bool, false, {"enable verbose output"}},
-            {"max-cluster-width", VariantType::Int, 1000 / 25, {"maximum time width of time clusters, in BC units"}},
-            {"peak-search-nbins", VariantType::Int, 5, {"number of time bins for the peak search algorithm (must be an odd number >= 3)"}},
-            {{"only-trackable"}, VariantType::Bool, false, {"remove digits for ROFs which are not trackable"}},
-            {"min-digits-per-rof", VariantType::Int, 0, {"minimum number of digits per ROF (below that threshold ROF is discarded)"}}}};
+            {"max-cluster-width", VariantType::Int, TimeClusterizerParam::Instance().maxClusterWidth, {"maximum time width of time clusters, in BC units"}},
+            {"peak-search-nbins", VariantType::Int, TimeClusterizerParam::Instance().peakSearchNbins, {"number of time bins for the peak search algorithm (must be an odd number >= 3)"}},
+            {{"only-trackable"}, VariantType::Bool, TimeClusterizerParam::Instance().onlyTrackable, {"remove digits for ROFs which are not trackable"}},
+            {"min-digits-per-rof", VariantType::Int, TimeClusterizerParam::Instance().minDigitsPerROF, {"minimum number of digits per ROF (below that threshold ROF is discarded)"}}}};
```

Comment:
Here again I would avoid mixing device options and configurable param and remove the options (and update the doc accordingly)